recipes/opam/meta.yaml

```diff
@@ -0,0 +1,39 @@
+#{% set version = "2.0.0-beta3" %}
+{% set unixsha256 = "6b560303316bc70099116ed3c5bfcc59928b9f8e10a99d994ea98ca6dad359d5" %}
+{% set osxsha256 = "6b560303316bc70099116ed3c5bfcc59928b9f8e10a99d994ea98ca6dad359d5" %}
+
+
+package:
+  name: opam
+  version: 2.0.0.beta3
+
+source:
+  fn: opam-2.0.0-beta3-x86_64-Linux
+  url: https://github.com/ocaml/opam/releases/download/2.0.0-beta3/opam-2.0.0-beta3-x86_64-Linux
+  sha256: {{ unixsha256 }}  # [unix]
+  fn: opam-2.0.0-beta3-x86_64-Darwin
+  url: https://github.com/ocaml/opam/releases/download/2.0.0-beta3/opam-2.0.0-beta3-x86_64-Darwin
+  sha256: {{ osxsha256 }}  # [osx]
+
+build:
+  number: 0
+  skip: True  # [win] 
+
+#requirements:
+#  run:
+#  - gmp >=5.0.1,<7
+
+test:
+  commands:
+    - python --version
```

Comment:
Is there any test of this package you could insert here?

Reply:
Yes @mwcraig I've changed this to `opam --version` in https://github.com/conda-forge/staged-recipes/pull/4191/commits/480aadb2aa7dfe18f5b90a89453be0d147c0c6f6 

I'm having a bit of trouble with the `build.sh` and `bld.bat` files

Reply:
Ohh, the build is failing in `CircleCI` . Here's the trace 

https://circleci.com/gh/conda-forge/staged-recipes/21261?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link